plugins/modules/cassandra_keyspace.py

```diff
@@ -268,6 +302,24 @@ def main():
     durable_writes = module.params['durable_writes']
     data_centres = module.params['data_centres']
 
+    if HAS_SSL_LIBRARY is False and ssl is True:
+        msg = ("This module requires the SSL python"
+               " library. You can probably install it with pip"
+               " install ssl.")
+        module.fail_json(msg=msg)
+
+    verify_mode = module.params['verify_mode']
+    ssl_verify_location = module.params['ssl_verify_location']
+
+    if verify_mode in ('CERT_REQUIRED', 'CERT_OPTIONAL') and module.params['ssl_verify_location'] == '':
```

Comment:
Could shorten module.params['ssl_verify_location'] to ssl_verify_location

Reply:
Also done in [b79fd7b](https://github.com/ansible-collections/community.cassandra/pull/230/commits/b79fd7beb470191d7b96b3ba64a0a3b34b3cbc2b)